pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -30,13 +30,23 @@ let
     # set to false if you want to control where to save the generated config
     # (e.g., in ~/.config/init.vim or project/.nvimrc)
     , wrapRc ? true
-    , neovimRcContent ? ""
+    , neovimRcContent ? null
+    , neovimLuaRcContent ? null
     # entry to load in packpath
     , packpathDirs
     , ...
   }:
+
+  # we can not have both, a lua and a vimL init file, at least one needs to be
+  # null
+  assert isNull neovimRcContent || isNull neovimLuaRcContent;
```

Comment:
This can be removed since user could set both, if only during the migration towards lua